tests/info_extraction/InfoScopeTest.py

```diff
@@ -0,0 +1,129 @@
+import os
+import unittest
+
+from coala_quickstart.info_extraction.Info import Info
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.InfoScope import InfoScope
+
+
+class InfoScopeTest(unittest.TestCase):
+
+    def setUp(self):
+
+        class DummyInfoExtractor(InfoExtractor):
+
+            def parse_file(self, file_content):
+                return file_content
+
+            def find_information(self, fname, parsed_file):
+                return [Info(
+                    fname,
+                    'Dummy information it is!')]
+
+        class NoInfoExtractor(InfoExtractor):
+
+            def parse_file(self, fname, file_content):
+                return file_content
+
+            def find_information(self, fname, parsed_file):
+                return []
+
+        class InfoA(Info):
+            description = 'Information A'
+            value_type = (str, int)
+            example_values = ['coala', 111]
+
```

Comment:
Why this 3 classes are part of `setUp` and not global ? Do we need to declare it fore all the tests separately ? If not, it can be global.